vllm/engine/async_llm_engine.py

```diff
@@ -924,6 +925,14 @@ async def get_model_config(self) -> ModelConfig:
         else:
             return self.engine.get_model_config()
 
+    async def get_parallel_config(self) -> ParallelConfig:
+        """Get the parallel configuration of the vLLM engine."""
+        if self.engine_use_ray:
```

Comment:
these `if`s are outta control, the ray engine should totally be a separate `VLLMBackend` 😉 

...a change for another day, or week

Reply:
Good idea